PrebidMobile/AdUnits/AdUnit.swift

```diff
@@ -387,6 +387,16 @@ public class AdUnit: NSObject, DispatcherDelegate {
         return adUnitConfig.gpid
     }
     
+    // MARK: Global ORTBObject
+    
+    public func setOrtbConfig(_ ortbObject: [String: Any]?) {
```

Comment:
The custom request should be set as a String. 

Publishers should just put the JSON string as a configuration option and not a dictionary.